### DISCUSSION TITLE
Powerdevil module is hard to understand

### DISCUSSION THREAD

**Discussion by jaanonim** [nix-community/plasma-manager#359] [Q&A] [answered]
Hi,

I'm finding powerdevil options to complicated to understend. Can sb explay how to configure those or give some exaple?

**Comment by HeitorAugustoLN** [comment, marked as answer]
Here is an example of my [powerdevil configuration](https://github.com/HeitorAugustoLN/nix-config/blob/main/home/heitor/features/desktop/plasma/powerdevil.nix).

And it reflects into these settings:

![Powerdevil settings](https://github.com/user-attachments/assets/7f22b342-20d4-455d-bb79-0a128413eadf).

You can see the description of each option in here: https://nix-community.github.io/plasma-manager/options.xhtml#opt-programs.plasma.powerdevil.AC.autoSuspend.action

**Reply by jaanonim** [reply]
wow I didn't know there is docs :-) 

Do you have any idea why I'm getting this error then:
```
 error: The option `home-manager.users.jaanonim.programs.plasma.powerdevil.AC.whenLaptopLidClosedActions' does not exist. Definition values:
       - In `/nix/store/nz18nd7i4fjja9jh4kdbz80kqy939y3q-source/common/home/optional/plasma.nix': "turnOffScreen"
```

my inputs:
```
Inputs:
├───home-manager: github:nix-community/home-manager/ec4c6928bbacc89cf10e9c959a7a47cbaad95344
│   └───nixpkgs follows input 'nixpkgs'
├───nixpkgs: github:nixos/nixpkgs/574d1eac1c200690e27b8eb4e24887f8df7ac27c
├───nixpkgs-stable: github:NixOS/nixpkgs/68e7dce0a6532e876980764167ad158174402c6f
├───plasma-manager: github:pjones/plasma-manager/3620206aec6d4d1ff0456ae1cf46e5b983121b6e
│   ├───home-manager follows input 'home-manager'
│   └───nixpkgs follows input 'nixpkgs'
```

**Reply by HeitorAugustoLN** [reply]
You are referencing an inexistent option, the actual option is `whenLaptopLidClosed`. `whenLaptopLidClosedActions` is an attribute set that is used by `whenLaptopLidClosed` option.

**Reply by jaanonim** [reply]
ow thanks so mutch <3
